Cargo.toml

```diff
@@ -117,6 +117,7 @@ tracing-subscriber = "0.3.17"
 trybuild = "1.0"
 ureq = "2.9.7"
 url = "2.5.2"
+uuid = { version = "1.10.0", features = ["serde"] }
```

Comment:
the root manifest should just define versions which can be inherited with no default feature activations unless with good reason (security or other similar constraint)

Reply:
The root manifest here defines the features that all usages of this crate need, i.e. serde. That is used everywhere that the crate is used, and is in place as a default.